openmc/mgxs/groups.py

```diff
@@ -299,3 +303,127 @@ def merge(self, other):
         # Assign merged edges to merged groups
         merged_groups.group_edges = list(merged_edges)
         return merged_groups
+
+
+class DelayedGroups(object):
+    """A delayed groups structure used for multi-delayed-group parameters.
+
+    Parameters
+    ----------
+    groups : Iterable of Int
+        The delayed groups
```

Comment:
Correct, it is just a list of delayed groups and allows the user to specify some list of the delayed groups. Since the number of delayed groups differs by cross section library and so users can tally to some subset of the delayed groups, I figured having this object would be best.